### ISSUE TITLE
[STU-169] Payments view is broken on Craft Pro 5.2.5

### ISSUE THREAD

**Issue by cookie10codes** [studioespresso/craft-mollie-payments#71]
Installed the version on the Craft Pro 5.1.10 for checking out the plugin. Now our project where I'm implementing the Mollie Payments, it errors out on the payments view.

There's an update within the CP.php helper, where it's checking for the indicator class to find a matching status, but it's probably not present on the Payment element, so the value can't be fetched.

It's this piece of code inside of the craftcms/cms/src/helpers/Cp.php line 801
```
$config['indicatorClass'] = match ($status) {
    'pending', 'off', 'suspended', 'expired', 'disabled', 'inactive' => $status,
    default => $config['color']->value,
};
```
![Screenshot 2024-07-04 at 14 06 22](https://github.com/studioespresso/craft-mollie-payments/assets/20947573/9f9fb3dd-96af-47fc-ab42-2fa6cab0f1b6)

<sub>[STU-169](https://linear.app/studio-espresso/issue/STU-169/payments-view-is-broken-on-craft-pro-525)</sub>

**Comment by janhenckens** [comment]
hey @cookie10codes, sorry about that! removing the status from the overview should fix that for now.

I'm working on a larger update (that includes support for Mollie Subscriptions), which also includes a fix this error, Should be out early next week

**Comment by cookie10codes** [comment]
No worries, happy to here it was already on your radar 👍🏼

**Comment by cookie10codes** [comment]
hi @janhenckens, do you know when the fix will be applied?

**Comment by janhenckens** [comment]
Hey @cookie10codes! Sorry about the delay, I have a client testing the subscription feature in a beta release ([5.1.0-beta.3](https://github.com/studioespresso/craft-mollie-payments/releases/tag/5.1.0-beta.3)) and that is taking a bit longer than I had anticipated in July.

I'm away next week and I don't want to rush the release so I'll try to get a patch update out of fix this in the meantime.

**Comment by cookie10codes** [comment]
@janhenckens only if it fits your schedule tho. Enjoy you time-off :-)

**Comment by janhenckens** [comment]
Should be fixed in 5.0.1 @cookie10codes :)
